lib/createJestRunner.ts

```diff
@@ -32,8 +33,14 @@ export default function createRunner<
 >(
   runPath: string,
   { getExtraOptions }: CreateRunnerOptions<ExtraOptions> = {},
-): typeof CallbackTestRunner {
-  return class BaseTestRunner extends CallbackTestRunner {
+) {
+  return class BaseTestRunner implements CallbackTestRunnerInterface {
+    #globalConfig: Config.GlobalConfig;
```

Comment:
TS is fine with `#`, but does not allow to have `private _globalConfig`: "Property '_globalConfig' of exported class expression may not be private or protected."

Hm.. Should Node 12 support be dropped before merging this change? https://github.com/jest-community/create-jest-runner/blob/445b564d702b085f12a470a4f696745581300df0/.github/workflows/nodejs.yml#L36-L37

Reply:
do we need to use `#` now, or is that just a refactoring?

Reply:
Can be `public` as well. Is that fine?

Reply:
Private is fine 🙂 this class isn't used by consumers anyways

Reply:
@mrazauskas landed https://github.com/jest-community/create-jest-runner/pull/135 fwiw